scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala

```diff
@@ -124,6 +138,7 @@ trait TypedDelimited[T] extends DelimitedScheme
   final override def sinkFields = fields
 }
 
+@deprecated("Use TypedText instead")
```

Comment:
Sounds good.

On Thursday, July 16, 2015, Sidharth Kapur <notifications@github.com> wrote:

> In scalding-core/src/main/scala/com/twitter/scalding/TypedDelimited.scala
> <https://github.com/twitter/scalding/pull/1356#discussion_r34813124>:
>
> > @@ -124,6 +138,7 @@ trait TypedDelimited[T] extends DelimitedScheme
> >    final override def sinkFields = fields
> >  }
> >
> > +@deprecated("Use TypedText instead")
>
> I believe this one should be FixedTypedText?
> since
> FixedTypedText extends FixedPathSource with TypedTextDelimited
> and
> FixedPathTypedDelimited extends FixedPathSource with TypedDelimited
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/twitter/scalding/pull/1356/files#r34813124>.
>


-- 
Oscar Boykin :: @posco :: http://twitter.com/posco